syntax = "proto3";

package dataflow;

message PoisonEvent {
  string reason = 1;
}